src/iop/highlights.c

```diff
@@ -281,11 +288,12 @@ int process_cl(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, cl_m
   // this works for bayer and X-Trans sensors
   if(visualizing)
   {
+    const float mclip = d->clip * highlights_clip_magics[d->mode];
```

Comment:
why use `mclip` here and `clip` in process_visualize() below?

Reply:
Arrgh, yes ...